src/Module.Server/Balancing/MatchBalancer.cs

```diff
@@ -362,8 +362,8 @@ private bool FindAndSwapUsers(GameMatch gameMatch, float sizeScaler)
         // If the team that has the least players is empty, we will do the swap with an empty clan group. A bit weird
         // but very practical for the rest of the algorithm to avoid null checks.
         ClanGroup emptyClanGroup = new(null);
-        ClanGroup weakClanGroupToSwap = weakClanGroupsTeam.FirstOrDefault() ?? emptyClanGroup;
-        ClanGroup strongClanGroupToSwap = strongClanGroupsTeam.LastOrDefault() ?? emptyClanGroup;
+        ClanGroup weakClanGroupToSwap = emptyClanGroup;
+        ClanGroup strongClanGroupToSwap = emptyClanGroup;
```

Comment:
Is the ClanGroup modified? Because both variables points to the same object

Reply:
explained